libfabric.spec

```diff
@@ -1,10 +1,12 @@
+%define libname libfabric1
```

Comment:
How is this going to affect that RedHat and Suse have different packages and package naming for eventual use of Distro provided packaging?

This looks like it is converting the RedHat packaging setup to the Suse names.

Reply:
Yeah.  I'm investigating RH packaging requirements on that front.

Reply:
After discussing library packaging standards with upstream, I'm going to revert the _libfabric$major_ for EL7.